src/app/public/modules/date-range-picker/types/date-range-calculator.ts

```diff
@@ -23,17 +23,33 @@ import {
 } from './date-range';
 
 export class SkyDateRangeCalculator {
+
+  /**
+   * Indicates the text displayed within the calculator select menu.
```

Comment:
```suggestion
   * Specifies the text to display in the calculator select menu.
```
What is the calculator select menu? I get that this property is providing text to display somewhere, but I don't understand where it appears.

Reply:
> What is the calculator select menu? I get that this property is providing text to display somewhere, but I don't understand where it appears.

Its the option/select menu that appears when consumers include the `sky-date-range-picker` component in their code.